### PR TITLE
added modules for nimble group and nimble user policy

### PR DESCRIPTION
**Pull request by ar-india**

### CODE REVIEW

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py [hunk 1]
```diff
@@ -161,7 +161,7 @@
   description: Returns the information collected from the HPE Nimble Storage array
   returned: always
   type: complex
-   "nimble_info": {
+  sample: {
```
Comment (ar-india) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_info.py:164]: changed the string to "sample" to support ansible-doc command to generate the example in return coloumn

**Review by ar-india — COMMENTED** [review]
please review

**Review by ar-india — COMMENTED** [review]
Gentle Reminder

**Review by suneeth51 — APPROVED** [review]
lgtm..

**Pull request merged**